service/src/test/resources/ehr/request/fhir/output/expected-xml-with-standalone-observations.xml

```diff
@@ -158,64 +146,110 @@ EMPTY REPORT</text>
 </component>
     </CompoundStatement>
 </component>
-    </CompoundStatement>
-</component>
-    </ehrComposition>
-</component>
-                <component typeCode="COMP">
-    <ehrComposition classCode="COMPOSITION" moodCode="EVN">
-        <id root="test-id-3" />
-        <code code="109341000000100" displayName="GP to GP communication transaction" codeSystem="2.16.840.1.113883.2.1.3.2.4.15"/>
-        <statusCode code="COMPLETE" />
+    <component typeCode="COMP" contextConductionInd="true">
```

Comment:
This change looks sus. Looking at the JSON, the DiagnosticReport has 1 Specimen, and zero test results.

We'd expect that the Mapper would generate 1 Specimen, and no Dummy, but it appears to be generating both.

Reply:
From looking at the XML, we can see two instances of "<specimen typeCode="SPC">", where the JSON suggests we should only have one, which agrees with what you've said.
However, that specific diagnostic report has one specimen and zero observations. A diagnostic report must have an observation. Therefore, a dummy observation is made, and linked to a dummy specimen. This accounts for the two specimens.

Reply:
Hypothesis as to what's going on:

1. When it grabs Specimens, it knows that there is one already.
2. When it grabs the Observations, it identifies there isn't one, so creates one.
3. Because this dummy Observation that has been created doesn't have a Specimen it also generates a Dummy specimen to go along with it.